src/util.ts

```diff
@@ -11,7 +11,7 @@ export async function shell(program: string, args: string[] = [], options: Spawn
   return new Promise((ok, ko) => {
     const child = spawn(program, args, { stdio: ['inherit', 'pipe', 'inherit'], ...options });
     const data = new Array<Buffer>();
-    child.stdout.on('data', chunk => data.push(chunk));
+    child.stdout?.on('data', chunk => data.push(chunk));
```

Comment:
This is due to a change in @types/node. Node.js docs says:

> If the child was spawned with stdio[1] set to anything other than 'pipe', then this will be null.

You can see above stdout is set to "pipe" so the usage of stdout is still correct.